session.go

```diff
@@ -296,6 +297,8 @@ func (session *Session) ReceiveWith(localClientID, remoteClientID string, buf []
 
 		if conn, ok := session.connections[connKey(localClientID, remoteClientID)]; ok {
 			conn.SendAck(packet.SequenceId)
+		} else {
+			return fmt.Errorf("can't find conn %v to send ACK", connKey(localClientID, remoteClientID))
```

Comment:
Better to create a new error type so one can check type outside. It's probably more important than printing the client id
```
ConnNotFoundError  = NewGenericError("connection not found", false, false)
```

Reply:
Yes, new error was created.